### ISSUE TITLE
Include a note about use of Google Chrome as the required browser.

### ISSUE THREAD

**Issue by nshamit** [LSFLK/MedicinesforLK#363]
- [ ] Include a note about use of Google Chrome as the required browser. There is a recaptcha error on Firefox and Safari (which is what the Red Cross tends to use more)

**Comment by jayasanka-sack** [comment]
@nipunsampath Is this a known error?

**Comment by nshamit** [comment]
@manuranga can we pass this on to the Asgardeo team for clarification?

**Comment by manuranga** [comment]
There is no recommended browser for Asgadeo. All three major browsers should work, if not we need to report to `asgardeo-help@wso2.com`.
I tested form Safari and Firefox multiple time I couldn't reproduce the issue after the original day.

**Comment by nshamit** [comment]
ack, we have a demo with the MOH/MSD today, so i'll report if we experience the same in this session. If not, we can close this issue.

**Comment by manuranga** [comment]
If issues occurs, screenshot with developer tools opened to network tab will help a lot. (note in safari this view need to be enabled form the preferences first)

**Comment by nipunsampath** [comment]
@jayasanka-sack It's not a known issue AFAIK. Apologies for the late reply.

**Comment by nshamit** [comment]
We have not seen this error since. Will close this and we can reopen if necessary.

**Comment by manuranga** [comment]
https://user-images.githubusercontent.com/1686124/184933226-07cbba91-a2ba-4434-8d66-5391b7c7ff48.mov

I am seeing this again in Safari
Asgadeo ref
```
x-azure-ref: 0I8f7YgAAAABPrQxkBMQpRYqIOkSb1gIFU0dFRURHRTA4MDcAOGM1ZTczZjQtNmZhZi00MzQ1LTg4YmYtYjEwZjBmODk1MGM3
```

**Comment by ratnajothy** [comment]
Would it be possible to enable the checkbox for `Preserve log` and then try exporting the HAR with network trace?

**Comment by manuranga** [comment]
Sorry no, it didn't work, Export button seems to be disabled.

**Comment by dinuka-rp** [comment]
Seems to be fixed by #430
